examples/wearable/UIComponents/navigationelements/fastscroll.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Examples files shuld be located in contents folder,
Link doesn't work in UIComponents app

Reply:
Done.